src/services/ledger/index.ts

```diff
@@ -72,23 +77,25 @@ export const LedgerService = (): ILedgerService => {
   }
 
   const getLiabilityTransactions = async (
-    liabilitiesAccountId: LiabilitiesAccountId,
+    walletId: WalletId,
   ): Promise<LedgerTransaction[] | LedgerError> => {
+    const liabilitiesWalletId = toLiabilitiesWalletId(walletId)
```

Comment:
I believe we deliberately didn't include these inside service methods previously to keep boundaries clear. @bodymindarts would have to confirm.

This would also be related to the function signature changes in `src/domain/ledger/index.types.d.ts`

Reply:
> I believe we deliberately didn't include these inside service methods previously to keep boundaries clear.

hmm. I think not including those function within ledger made the boundary not clear in my mind. because now we have added to to the conversion of `walletId` to `liabilitiesWalletId` everywhere in the codebase, where `liabilitiesWalletId` is a notion that should only concern the ledger IMO.

Reply:
Marking this as resolved based on our discussion about making the Ledger sub-domain non-generic (i.e. aware of core domain concepts)